### PR TITLE
 ComputedValues is now ServoComputedValues

### PR DESCRIPTION
**Pull request by perlun**
This is the first part of #10185. More to follow. I have built this locally with both servo and geckolib without errors; let's see if it succeeds on all platforms as well.

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/10210)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,7 @@
 /ports/android/libs
 /ports/android/local.properties
 /ports/android/obj
+/ports/geckolib/target
```
Comment (bholley) [inline .gitignore]: What is this about?

Comment (perlun) [inline .gitignore]: I got some files there when I was building geckolib, or so I _thought_ - rebuilding now doesn't do it. I think it must have been some background compilation done in Atom while I was editing the files. Reverting that part now, thanks for seeing it.

**Comment by perlun** [comment]
Thanks Simon! Will try to continue with the remaining renames asap.

**Pull request merged**